<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> Published Research | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Published Research"/>
<meta name="twitter:description" content="Other projects Genomic Privacy Genomic privacy and data sharing are issues of relevance for the whole scientific community. Protecting the privacy of individuals who participate in a study has always been a top priority and it has been widely assumed that publishing summary results did not jeopardize privacy. In 2008, Homer et al found that in the case of genome wide association studies (GWAS), summary results such as allele frequencies for a large number of genetic variants can reveal whether a person participated in a study and the disease status of the individual."/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/">Home</a>
      </li>
      
      <li>
        <a href="/project/">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container single">
<main>

<div class="article-meta">
<h1><span class="title">Published Research</span></h1>


<p class="terms">
  
  
  
  
  Tags: <a href="/tags/research">research</a> 
  
  
</p>
</div>

<div class="article">
<h2 id="other-projects">Other projects</h2>
<h3 id="genomic-privacy">Genomic Privacy</h3>
<p>Genomic privacy and data sharing are issues of relevance for the whole scientific community. Protecting the privacy of individuals who participate in a study has always been a top priority and it has been widely assumed that publishing summary results did not jeopardize privacy. In 2008, Homer et al found that in the case of genome wide association studies (GWAS), summary results such as allele frequencies for a large number of genetic variants can reveal whether a person participated in a study and the disease status of the individual. These results forced the NIH to withdraw most of the public access to GWAS study results. We were interested in sharing results from quantitative traits such as gene expression phenotypes, which provide critical information on the regulatory role of genetic variants. The question here was whether publishing regression coefficients from GWAS would also allow re-identification. Dr. Im proved mathematically that re-identification based on regression coefficients was possible, provided an explicit method and computed its theoretical power as a function of sample size, number of markers, and false positive rate. In fact, She found that even the sign of the regression coefficients was enough to reveal a person’s participation. This is exemplary publication #1, which was published in American Journal of Human Genetics (Im et al, 2012). Given its wide public relevance, this paper was featured on the University of Chicago Hospitals Science <a href="http://sciencelife.uchospitals.edu/2012/05/29/a-crack-in-the-safe-of-genomic-studies">blog - click here</a> and has been routinely cited by papers on genomic privacy.</p>
<h3 id="poly-omic-prediction-of-complex-traits">Poly-Omic Prediction of Complex Traits.</h3>
<p>Prediction of disease risk or treatment response is one of the pillars of personalized medicine. Although genome-wide association studies have discovered thousands of well-replicated polymorphisms associated with a broad spectrum of complex traits, the combined predictive power of these associations for any given trait is generally too low to be of clinical relevance. To address these issues, We proposed a systems approach to complex trait prediction, which leverages and integrates similarity in genetic and other high throughput molecular traits (omic data). The approach translates the omic similarity into phenotypic similarity using a method called Kriging, commonly used in Geostatistics. My method called OmicKriging emphasizes the use of a wide variety of systems-level data, such as those increasingly made available by comprehensive surveys of the genome, transcriptome and epigenome, for complex trait prediction. Application to clinical and cellular phenotypes shows the advantages of integrating multiple omic data in a collective manner. This was published in Genetic Epidemiology (Wheeler et al 2014). This paper has been chosen for the Robert C. Elston Best Paper Award by the International Genetic Epidemiology Society (IGES) to be awarded at the IGES meeting in October 2015.</p>

</div>
</main>

<section class="appendix">







<div>
  <div class="side side-left"><h3>Reuse</h3></div>
  Text and figures are licensed under <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution CC BY 4.0</a>. The source code is licensed under MIT.
</div>



<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepageproject/other-projects.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>




</section>



<nav class="post-nav">
  <span class="nav-next">&larr; <a href="/project/published-research/" title="Next post (older)">Published Research</a></span>
  &hercon;
  <span class="nav-prev"></span>
</nav>




<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//hakyimlab.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    var t = d.getElementById('disqus_thread');
    var b = false, l = function(scroll) {
      if (b) return;
      (d.head || d.body).appendChild(s); b = true;
      if (scroll) t.scrollIntoView();
    }
    s.onerror = function(e) {
      if (sessionStorage.getItem('failure-note')) return;
      t.innerText = 'Sorry, but you cannot make comments because Disqus failed to load for some reason. It is known to be blocked in China. If you are sure it is not blocked in your region, please refresh the page. 中国大陆地区读者需要翻墙才能发表评论。';
      t.style.border = '1px dashed';
      t.style.padding = '.5em';
      t.style.background = 'lightyellow';
      sessionStorage.setItem('failure-note', true);
    };
    
    if (location.hash.match(/^#comment-[0-9]+$/)) return l(true);
    var c = function() {
      if (b) return;
      var rect = t.getBoundingClientRect();
      if (rect.top < window.innerHeight && rect.bottom >= 0) l();
    };
    window.addEventListener('load', c);
    d.addEventListener('scroll', c);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>


</div>


  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

