<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> Published Research | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Published Research"/>
<meta name="twitter:description" content="Published research PrediXcan, a regulatory mechanism driven gene discovery tool Although investments in genomic studies of complex diseases made possible the discovery of thousands of variants robustly associated with these traits, the translation of these discoveries into actionable targets has been hampered by the lack of a mechanistic understanding on how genome variation relates to phenotype. Moreover, it has been widely shown that a substantial portion of the genetic control of complex traits is exerted through the regulation of gene expression traits, but effective methods to fully harness this mechanism were not available."/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/">Home</a>
      </li>
      
      <li>
        <a href="/project/">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container single">
<main>

<div class="article-meta">
<h1><span class="title">Published Research</span></h1>


<p class="terms">
  
  
  
  
  Tags: <a href="/tags/research">research</a> 
  
  
</p>
</div>

<div class="article">
<h2 id="published-research">Published research</h2>
<h3 id="predixcan-a-regulatory-mechanism-driven-gene-discovery-tool">PrediXcan, a regulatory mechanism driven gene discovery tool</h3>
<p>Although investments in genomic studies of complex diseases made possible the discovery of thousands of variants robustly associated with these traits, the translation of these discoveries into actionable targets has been hampered by the lack of a mechanistic understanding on how genome variation relates to phenotype. Moreover, it has been widely shown that a substantial portion of the genetic control of complex traits is exerted through the regulation of gene expression traits, but effective methods to fully harness this mechanism were not available. To address these challenges, we developed a gene-based test —PrediXcan— that directly tests this regulatory mechanism and substantially improves power relative to single variant tests and other gene-based tests. PrediXcan is inherently mechanistic and provides directionality, highlighting its potential utility in identifying novel targets for therapy. PrediXcan was the first of the class of methods known currently as TWAS (transcriptome-wide association studies). More details can be found <a href="https://uchicago.box.com/shared/static/ilqi3c12j5z8gu6puiiemumai0vgl6h2.pdf">in this paper</a>. The following figure shows a schematic description of the method.</p>
<table>
<thead>
<tr>
<th style="text-align:center"><img src="https://uchicago.box.com/shared/static/d3ab2xn2ivrj3yfo9tkkeo3qrfew4k4k.png" alt=""></th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center"><em>PrediXcan tests the mediating effect of a target gene on disease risk.</em> The left panel shows the components of the expression of a gene. PrediXcan tests the association between the genetically regulated component and the complex trait and is not affected by the reverse effects of the disease on the expression. The mechanistic model behind PrediXcan is shown on the right panel. The genetic profile of an individual determines a baseline gene expression level (genetically determined expression), which is further modified by environment and other factors. For deleterious causal genes this final level determines the liability to disease which can lead to disease if it surpasses certain threshold.</td>
</tr>
</tbody>
</table>
<h3 id="scaling-up-predixcan-to-integrate-large-scale-data">Scaling up PrediXcan to integrate large-scale data</h3>
<p>Genetic studies have discovered many thousands of locations on chromosomes that are associated with health or disease, but the majority of these fall outside of protein-encoding regions, and so it is difficult to understand just how they exert their influences. PrediXcan leverages large scale DNA, observable trait, and gene expression datasets, as well as machine learning, to infer from these discoveries which genes are actually influenced and whether an up or downregulation of the gene is beneficial. Despite the appeal and success of this approach, which directly identifies candidate drug targets, accumulating experience revealed that data from over a million individuals would be required in many cases. We therefore developed an approach that bypasses the need to use individual level data and takes advantage of summary results, to vastly expand the applicability and power of the original PrediXcan method. The new approach, called S-PrediXcan, allows to find potential target genes for modulating thousands of traits across a broad set of human tissues. These results are now publicly available and have gathered users across the globe and led to funded collaborations. Most of the associations proved to be tissue-specific, suggesting context specificity of the trait etiology. Significant associations in unexpected tissues also underscored the need for an agnostic scanning of multiple contexts to improve the ability to detect causal regulatory mechanisms. Monogenic disease genes were enriched among significant associations for related traits, suggesting that smaller alterations of these genes may cause a spectrum of milder phenotypes. More details can be found <a href="https://www.nature.com/articles/s41467-018-03621-1">in this paper</a></p>
<h3 id="genetic-architecture-of-expression-traits-across-tissues">Genetic architecture of expression traits across tissues</h3>
<p>We investigated the genetic architecture of gene expression traits, which are important intermediate processes to understand the mechanisms by which genetic variation affect complex diseases and traits. We found that in contrast to the polygenic nature of complex diseases, most of the variation of gene expression traits is driven by a small number of variants (eQTLs). In addition to providing insight into the biology of the transcriptome, this finding has practical implications for the best approaches to predict gene expression traits. More details can be found <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007586">in this paper</a></p>
<h3 id="boosting-the-power-to-identify-target-genes-by-exploiting-the-tissue-sharing-of-gene-expression-regulation">Boosting the power to identify target genes by exploiting the tissue sharing of gene expression regulation</h3>
<p>An unexpected finding from the GTEx consortium (large scale transcriptome study of 900 organ donors with whole body tissue sampled across 49 tissues) was the widespread sharing of genetic regulation of expression traits across tissues. We estimated that more than half of the eQTLs (variants associated with expression levels) are active across all tissues. One implication of this is that eQTL-based studies will have little specificity to narrow down the tissue where disease initiates. However, one can take advantage of this lack of specificity and consider the different tissue panels as independent experiments and aggregate the information across all of them. We developed MultiXcan to implement this idea and show its effectiveness. More details can be found <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889">in this paper</a></p>
<h3 id="application-to-childhood-and-adult-onset-asthma-data-in-the-uk-biobank">Application to childhood and adult onset asthma data in the UK Biobank</h3>
<p>Regardless of how sophisticated they may be, analytical methods only matter if they can answer relevant scientific questions. We teamed up with Carole Ober and Dan Nicolae, experts in asthma genetics, to examine the shared and distinct genetic factors of childhood and adult onset asthma risk applying state of the art methods implemented in our lab’s analytical pipeline. For the first time, data on age of onset of asthma became available through the UK Biobank for a sufficiently large number of individuals. We seized the opportunity to identify novel genetic factors that are specific to childhood onset cases as well as those shared with adult onset cases. We found that genetic factors of adult onset asthma are a subset of childhood onset asthma with overall smaller effects suggesting a larger environmental component in adult cases. Tissue enrichment analysis suggested the role of allergy and epithelial barrier disfunction in childhood onset asthma and lung involvement in adult onset asthma. Both types shared immune components. More details can be found <a href="https://uchicago.box.com/shared/static/oxziuex5lpjzrg3oguobnpvk1amehhz0.pdf">in this paper</a></p>

</div>
</main>

<section class="appendix">







<div>
  <div class="side side-left"><h3>Reuse</h3></div>
  Text and figures are licensed under <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution CC BY 4.0</a>. The source code is licensed under MIT.
</div>



<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepageproject/published-research.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>




</section>



<nav class="post-nav">
  <span class="nav-next">&larr; <a href="/project/ongoing-research/" title="Next post (older)">Ongoing Research</a></span>
  &hercon;
  <span class="nav-prev"><a href="/project/other-projects/" title="Previous post (newer)">Published Research</a> &rarr;</span>
</nav>




<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//hakyimlab.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    var t = d.getElementById('disqus_thread');
    var b = false, l = function(scroll) {
      if (b) return;
      (d.head || d.body).appendChild(s); b = true;
      if (scroll) t.scrollIntoView();
    }
    s.onerror = function(e) {
      if (sessionStorage.getItem('failure-note')) return;
      t.innerText = 'Sorry, but you cannot make comments because Disqus failed to load for some reason. It is known to be blocked in China. If you are sure it is not blocked in your region, please refresh the page. 中国大陆地区读者需要翻墙才能发表评论。';
      t.style.border = '1px dashed';
      t.style.padding = '.5em';
      t.style.background = 'lightyellow';
      sessionStorage.setItem('failure-note', true);
    };
    
    if (location.hash.match(/^#comment-[0-9]+$/)) return l(true);
    var c = function() {
      if (b) return;
      var rect = t.getBoundingClientRect();
      if (rect.top < window.innerHeight && rect.bottom >= 0) l();
    };
    window.addEventListener('load', c);
    d.addEventListener('scroll', c);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>


</div>


  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

