<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.46" />
  <meta name="author" content="Hae Kyung Im">
  <meta name="description" content="Assistant Professor - Genetic Medicine - ">

  
  
  
    
  
  
    
    
    <link rel="stylesheet" href="/css/highlight.min.css">
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-61894206-7', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  <link rel="alternate" href="http://hakyimlab.org/index.xml" type="application/rss+xml" title="IM-LAB">
  <link rel="feed" href="http://hakyimlab.org/index.xml" type="application/rss+xml" title="IM-LAB">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="http://hakyimlab.org/post/2019/predictdb/">

  

  <title>PredictDB: Transcriptome Prediction Model Repository | IM-LAB</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">IM-LAB</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/#home">
            
            <span>Home</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#people">
            
            <span>People</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#posts">
            
            <span>News</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#publications">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#talks">
            
            <span>Talks</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#projects">
            
            <span>Software</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">PredictDB: Transcriptome Prediction Model Repository</h1>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2020-01-07 09:18:21 -0600 CST" itemprop="datePublished">
      Tue, Jan 7, 2020
    </time>
  </span>

  
  
  
  

  
  
  
  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=http%3a%2f%2fhakyimlab.org%2fpost%2f2019%2fpredictdb%2f"
         target="_blank">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=PredictDB%3a%20Transcriptome%20Prediction%20Model%20Repository&amp;url=http%3a%2f%2fhakyimlab.org%2fpost%2f2019%2fpredictdb%2f"
         target="_blank">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=http%3a%2f%2fhakyimlab.org%2fpost%2f2019%2fpredictdb%2f&amp;title=PredictDB%3a%20Transcriptome%20Prediction%20Model%20Repository"
         target="_blank">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=http%3a%2f%2fhakyimlab.org%2fpost%2f2019%2fpredictdb%2f&amp;title=PredictDB%3a%20Transcriptome%20Prediction%20Model%20Repository"
         target="_blank">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=PredictDB%3a%20Transcriptome%20Prediction%20Model%20Repository&amp;body=http%3a%2f%2fhakyimlab.org%2fpost%2f2019%2fpredictdb%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>

    <div class="article-style" itemprop="articleBody">
      

<h1 id="predictdb">Predictdb</h1>

<p>Here you can find transcriptome prediction models for the PrediXcan family of methods: S-PrediXcan, MultiXcan and S-MultiXcan.
 <code>.db</code> files are prediction models, usable by all methods.
 <code>.txt.gz</code> files are compilations of LD reference for summary-based methods (<code>S-</code> prefix).
 S-PrediXcan is meant to use the single-tissue LD reference files (&ldquo;covariances&rdquo;) appropriate to each model.
 S-MultiXcan uses single-tissue prediction models and a cross-tissue LD reference.</p>

<h1 id="gtex-v8-models-on-eqtl-and-sqtl">GTEx v8 models on eQTL and sQTL</h1>

<p>We have produced different families of prediction models for sQTL and eQTL,
using several prediction strategies, on GTEx v8 release data.</p>

<p>We recommend MASHR-based models below.
Elastic Net-based are a safe, robust alternative with decreased power.</p>

<h2 id="mashr-based-models">MASHR-based models</h2>

<p>Expression and splicing prediction models with LD reference data are available in this
<a href="https://zenodo.org/record/3518299" target="_blank">Zenodo repository</a>.</p>

<p>Files:
* <a href="https://zenodo.org/record/3518299/files/mashr_eqtl.tar?download=1" target="_blank">mashr_eqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s support on expression
* <a href="https://zenodo.org/record/3518299/files/mashr_sqtl.tar?download=1" target="_blank">mashr_sqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s support on splicing
* <a href="https://zenodo.org/record/3518299/files/gtex_v8_expression_mashr_snp_smultixcan_covariance.txt.gz?download=1" target="_blank">gtex_v8_expression_mashr_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan expression&rsquo;s LD reference
* <a href="https://zenodo.org/record/3518299/files/gtex_v8_splicing_mashr_snp_smultixcan_covariance.txt.gz?download=1" target="_blank">gtex_v8_splicing_mashr_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan splicing&rsquo;s LD reference</p>

<p><strong>Warning</strong>: these models are based on fine-mapped variants that may occasionally be absent
in a tipical GWAS, and frequently absent in older GWAS.
We have tools to address this, presented
<a href="https://github.com/hakyimlab/MetaXcan/wiki/Best-practices-for-integrating-GWAS-and-GTEX-v8-transcriptome-prediction-models" target="_blank">here</a>.
A tutorial is available <a href="https://github.com/hakyimlab/MetaXcan/wiki/Tutorial:-GTEx-v8-MASH-models-integration-with-a-Coronary-Artery-Disease-GWAS" target="_blank">here</a>.</p>

<p><strong>Acknowledging these models</strong>: If you use these models in your research, please cite:
* <a href="https://www.biorxiv.org/content/10.1101/814350v1" target="_blank">&ldquo;Widespread dose-dependent effects of RNA expression and splicing on complex diseases and traits&rdquo;</a>, Barbeira et al, 2019, preprint
* <a href="https://www.nature.com/articles/ng.3367" target="_blank">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics
* <a href="https://www.nature.com/articles/s41467-018-03621-1" target="_blank">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</p>

<p>If you use S-MultiXcan, we ask you to cite:
* <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889" target="_blank">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</p>

<h2 id="elastic-net">Elastic Net</h2>

<p>Expression and splicing prediction models with LD references data are available in this
<a href="https://zenodo.org/record/3519321#.XfqJz9l7m90" target="_blank">Zenodo repository</a>.</p>

<p>Files:</p>

<ul>
<li><a href="https://zenodo.org/record/3519321/files/elastic_net_eqtl.tar?download=1" target="_blank">elastic_net_eqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s support on expression</li>
<li><a href="https://zenodo.org/record/3519321/files/elastic_net_sqtl.tar?download=1" target="_blank">elastic_net_sqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s on splicing</li>
<li><a href="https://zenodo.org/record/3519321/files/gtex_v8_expression_elastic_net_snp_smultixcan_covariance.txt.gz?download=1" target="_blank">gtex_v8_expression_elastic_net_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan expression&rsquo;s LD reference</li>
<li><a href="https://zenodo.org/record/3519321/files/gtex_v8_splicing_elastic_net_snp_smultixcan_covariance.txt.gz?download=1" target="_blank">gtex_v8_splicing_elastic_net_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan splicing&rsquo;s LD reference</li>
</ul>

<p><strong>Acknowledging these models</strong> : If you use these models in your research, we ask you to cite:
* <a href="https://www.biorxiv.org/content/10.1101/787903v1" target="_blank">&ldquo;The GTEx Consortium atlas of genetic regulatory effects across human tissues&rdquo;</a>, Aguet et al, 2019, preprint
* <a href="https://www.nature.com/articles/ng.3367" target="_blank">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics
* <a href="https://www.nature.com/articles/s41467-018-03621-1" target="_blank">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</p>

<p>If you use S-MultiXcan, we ask you to cite:
* <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889" target="_blank">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</p>

<h1 id="gtex-v7-expression-models">GTEx v7 Expression models</h1>

<p>Expression prediction models with LD reference data ar available in this
<a href="https://zenodo.org/record/3572799" target="_blank">Zenodo repository</a>.
The underlying algorithm is Elastic Net.</p>

<ul>
<li><a href="https://zenodo.org/record/3572799/files/GTEx-V7_HapMap-2017-11-29.tar.gz?download=1" target="_blank">GTEx-V7_HapMap-2017-11-29.tar.gz</a>: Expression prediction models and LD reference on single tissues.</li>
<li><a href="https://zenodo.org/record/3572799/files/snp_covariance_v7.txt.gz?download=1" target="_blank">snp_covariance_v7.txt.gz</a>: S-MultiXcan&rsquo;s LD compilation</li>
</ul>

<p>Additional support information and details are available in the Zenodo repository.</p>

<p><strong>Acknowledging these models</strong>: If you use these models in your research, we ask you to cite:
* <a href="https://www.nature.com/articles/ng.3367" target="_blank">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics
* <a href="https://www.nature.com/articles/s41467-018-03621-1" target="_blank">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</p>

<p>If you use S-MultiXcan, we ask you to cite:
* <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889" target="_blank">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</p>

<h1 id="gtex-v6-expression-models">GTEx v6 Expression models</h1>

<p>Expression prediction models with LD reference data are available in this
<a href="https://zenodo.org/record/3572842#.XfuyLNl7m90" target="_blank">Zenodo repository</a>.</p>

<ul>
<li><a href="https://zenodo.org/record/3572842/files/GTEx-V6p-1KG-2016-11-16.tar.gz?download=1" target="_blank">GTEx-V6p-1KG-2016-11-16.tar.gz</a>: Expression prediction models and LD reference, using variants defined in 1000 Genomes</li>
<li><a href="https://zenodo.org/record/3572842/files/GTEx-V6p-HapMap-2016-09-08.tar.gz?download=1" target="_blank">GTEx-V6p-HapMap-2016-09-08.tar.gz</a>: Expression prediction models and LD reference, using variants defined in Hapmap CEU track</li>
<li><a href="https://zenodo.org/record/3572842/files/snp_covariance_v6p.txt.gz?download=1" target="_blank">snp_covariance_v6p.txt.gz</a>: S-MultiXcan LD reference for HapMap models</li>
</ul>

<p>Additional support information and details are available in the Zenodo repository.</p>

<p><strong>Acknowledging these models</strong>: If you use these models in your research, we ask you to cite:
* <a href="https://www.nature.com/articles/ng.3367" target="_blank">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics
* <a href="https://www.nature.com/articles/s41467-018-03621-1" target="_blank">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</p>

<p>If you use S-MultiXcan, we ask you to cite:
* <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889" target="_blank">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</p>

<h1 id="models-from-collaborators-and-other-sources">Models from collaborators and other sources:</h1>

<h2 id="mesa-models">MESA models</h2>

<p>Single-tissue expression prediction models with LD reference data are available in this
<a href="https://zenodo.org/record/3572815#.XfqOstl7m90" target="_blank">Zenodo repository</a>.
The underlying algorithm is Elastic Net on MESA multi-ethnic cohort.</p>

<p>These models were presented in <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007586" target="_blank">&ldquo;Genetic architecture of gene expression traits across diverse populations&rdquo;</a>, Mogil et al, 2018, PLOS Genetics.
Please cite if you find these useful.</p>

<h2 id="commonmind-consortium">CommonMind consortium</h2>

<p>Single-tissue expression prediction models with LD reference data are available in this
<a href="https://github.com/laurahuckins/CMC_DLPFC_prediXcan" target="_blank">GitHub repository</a>.
The underlying algorithm is Elastic Net.</p>

<p>These models were presented in <a href="https://www.nature.com/articles/s41588-019-0364-4" target="_blank">&ldquo;Gene expression imputation across multiple brain regions provides insights into schizophrenia risk</a>,
Huckins et al, 2019, Nature Genetics. Please cite if you find these useful.</p>

<h2 id="epixcan-models">EpiXcan Models</h2>

<p>Expression prediction models with LD reference data are available in this <a href="https://zhangw17.u.hpc.mssm.edu/epixcan/about.php#dl" target="_blank">website</a>.
The models were trained on Common Mind Consortium, GTEx, and STARNET consortiums.
The underlying algorithm is Elastic Net, informed by epigenetic data.</p>

<p>These models were presented in <a href="https://www.nature.com/articles/s41467-019-11874-7" target="_blank">&ldquo;Integrative transcriptome imputation reveals tissue-specific and shared biological mechanisms mediating susceptibility to complex traits&rdquo;</a>,
 Zhaneg et al, 2019, Nature Communications. Please cite if you find these useful.</p>

    </div>
  </div>

</article>




<div class="container">
  <nav>
  <ul class="pager">
    
    <li class="previous"><a href="http://hakyimlab.org/post/2019/gtex-v8-model-release/"><span
      aria-hidden="true">&larr;</span> GTEx V8 Model Release</a></li>
    

    
  </ul>
</nav>

</div>

<div class="article-container">
  
<section id="comments">
  <div id="disqus_thread"></div>
<script>
    var disqus_config = function () {
    
    
    
    };
    (function() {
        if (["localhost", "127.0.0.1"].indexOf(window.location.hostname) != -1) {
            document.getElementById('disqus_thread').innerHTML = 'Disqus comments not available by default when the website is previewed locally.';
            return;
        }
        var d = document, s = d.createElement('script'); s.async = true;
        s.src = '//' + "HakyImLab" + '.disqus.com/embed.js';
        s.setAttribute('data-timestamp', +new Date());
        (d.head || d.body).appendChild(s);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
<a href="https://disqus.com" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
</section>


</div>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2017 Hae Kyung Im &middot; 

      Powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha512-jGsMH83oKe9asCpkOVkBnUrDDTp8wl+adkB2D+//JtlxO4SrLoJdhbOysIFQJloQFD+C4Fl1rMsQZF76JjV0eQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.2/imagesloaded.pkgd.min.js" integrity="sha512-iHzEu7GbSc705hE2skyH6/AlTpOfBmkx7nUqTLGzPYR+C1tRaItbRlJ7hT/D3YQ9SV0fqLKzp4XY9wKulTBGTw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/TweenMax.min.js" integrity="sha512-Z5heTz36xTemt1TbtbfXtTq5lMfYnOkXM2/eWcTTiLU01+Sw4ku1i7vScDc8fWhrP2abz9GQzgKH5NGBLoYlAw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/plugins/ScrollToPlugin.min.js" integrity="sha512-CDeU7pRtkPX6XJtF/gcFWlEwyaX7mcAp5sO3VIu/ylsdR74wEw4wmBpD5yYTrmMAiAboi9thyBUr1vXRPA7t0Q==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

