<!DOCTYPE html>
<html lang="en-us">
  <head>
	<meta name="generator" content="Hugo 0.78.2" />
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> Home | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Home"/>
<meta name="twitter:description" content="ImLab website, built with Hugo and blogdown with the hugo-prose theme.
"/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/" class="active">Home</a>
      </li>
      
      <li>
        <a href="/project/">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container list">
<main>



<section class="article-list">
<div><h1 id="welcome-to-the-im-labs-home-page">Welcome to the IM Lab&rsquo;s home page</h1>
<p>We develop quantitative and computational methods and tools to sift through the vast amounts of genomic and other high dimensional data with the goal of making discoveries that can be translated to improve human health.</p>
</div>
</section>




<section class="article-list">
  <table>
  <tbody>
  <tr>
  
  <td><h2 id="research">Research</h2>
<p>The overarching goal of our research program is to develop computationally efficient and statistically sound methods to sift through the vast amounts of genomic and other high dimensional data with the goal of making discoveries that can be translated to improve human health.</p>
<p>To achieve this goal, we build and maintain an analytic infrastructure, i.e. a dry lab. This includes statistical and computational methods, pipelines and workflows, streamlined access to big data, and a research software engineering, bioinformatic and analysis team. This infrastructure constitutes a powerful instrument that gives us a unique vantage point from which discoveries can be made.</p>
<p>Check <a href="project/">this link</a> for some highlights of the methods we have developed and future directions of the lab.</p>
</td>
  
  </tr>
  </tbody>
  </table>
</section>

<section class="article-list">
  <table>
  <tbody>
  <tr>
  
  <td><h2 id="the-team">The Team</h2>
<p><img src="https://storage.googleapis.com/imlab-public/web-pictures/2020/imlab-zoom-2020-Sept-Oct-Collage.png" alt=""></p>
<ul>
<li>
<p>Hae Kyung Im, PhD - <a href="#about">Bio</a> - <a href="https://orcid.org/0000-0003-0333-5685">ORCID</a></p>
</li>
<li>
<p>Yanyu Liang, (GGSB Grad Student) - <a href="https://github.com/liangyy">Bio</a></p>
</li>
<li>
<p>Festus Nyasimi, (MS Bioinformatics) - <a href="https://github.com/Fnyasimi">Bio</a></p>
</li>
<li>
<p>Natasha Santhanam (Recent graduate) - <a href="https://github.com/natashasanthanam">Bio</a></p>
</li>
<li>
<p>Sheila Rajagopal (Fellow)</p>
</li>
</ul>
<h2 id="part-time-interns-externs">part time, interns, externs</h2>
<ul>
<li>
<p>Ian Waters, (PhD, Medical Student) - <a href="https://github.com/watersia">Bio</a></p>
</li>
<li>
<p>Wanhao Chi, (PhD) - <a href="https://github.com/wchi1">Bio</a></p>
</li>
<li>
<p>David Gonzalez-Velez, (Undergrad) - <a href="https://github.com/DGV98">Bio</a></p>
</li>
<li>
<p>Sabrina Mi, (Undergrad) - <a href="https://github.com/sabrina-mi">Bio</a></p>
</li>
<li>
<p>Julius Kim, (Undergrad) - <a href="https://github.com/theMechanic23/homepage">Bio</a></p>
</li>
</ul>
</td>
  
  </tr>
  </tbody>
  </table>
</section>

<section class="article-list">
  <table>
  <tbody>
  <tr>
  
  <td><h2 id="publications">Publications</h2>
<p>Find the full list of our publications courtesy of Google Scholar <a href="https://scholar.google.com/citations?hl=en&amp;user=1QD4sIcAAAAJ&amp;view_op=list_works&amp;sortby=pubdate">following this link here</a></p>
</td>
  
  </tr>
  </tbody>
  </table>
</section>

<section class="article-list">
  <table>
  <tbody>
  <tr>
  
  <td><ul>
<li>
<p>Rodrigo Bonazzola, MS - <a href="http://www.cistib.org/index.php/people/students/rodrigo-bonazzola">Currently PhD Student at the University of Leeds</a> <a href="https://github.com/rbonazzola">Github</a></p>
</li>
<li>
<p>Jiamao Zheng, PhD - <a href="https://github.com/jiamaozheng">Bio</a></p>
</li>
<li>
<p>Scott Dickinson, MS - <a href="https://github.com/ScottPDickinson">Bio</a></p>
</li>
<li>
<p>Padma Sheila Rajagopal, MD - <a href="https://github.com/parajago">Bio</a></p>
</li>
<li>
<p>Josh Jiang (Zhuoxun) - <a href="https://www.linkedin.com/in/zhuoxun-jiang-63499767/">Bio</a></p>
</li>
<li>
<p>Bayle Smith-Salzberg - <a href="https://www.linkedin.com/in/bayle-smith-salzberg-59b033154/">Bio</a></p>
</li>
<li>
<p>Owen Melia - <a href="https://github.com/meliao">Bio</a></p>
</li>
<li>
<p>Alvaro Barbeira, MS - <a href="https://github.com/heroico">Bio</a></p>
</li>
<li>
<p>Milton Pividori, PhD - <a href="https://github.com/miltondp">Bio</a> - <a href="https://orcid.org/0000-0002-3035-4403">ORCID</a></p>
</li>
</ul>
</td>
  
  </tr>
  </tbody>
  </table>
</section>

<section class="article-list">
  <table>
  <tbody>
  <tr>
  
  <td><h2 id="contact">Contact</h2>
<p>Email haky at uchicago dot edu</p>
<p>@hakyim</p>
</td>
  
  </tr>
  </tbody>
  </table>
</section>









  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2020/12/17/genomic-data-scientist-opening/">Genomic Data Scientist Opening</a></h1>
    <div>
      <span>2020-12-17</span>
      <span>Haky Im</span>
    </div>
    <div class="summary">
      
      
      
      
      [&hellip;] We are looking to hire a life-long learner interested in developing statistical and computational tools to sift through large amounts of data with the ultimate goal of making discoveries that can make a difference in the health of people. Methods and resources created in our lab are being used by researchers around the globe. We continue innovating and providing user-friendly and statistically efficient tools to get the most out of the ever-increasing amount of data. We partner with large consortia and biomedical researchers to find the most pressing questions in the field and apply our statistical expertise to develop reliable and efficient approaches to answer them.

      <a href="/post/2020/12/17/genomic-data-scientist-opening/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2020/11/10/r-markdown-demo/">R Markdown Demo</a></h1>
    <div>
      <span>2020-11-10</span>
      <span>Yihui Xie</span>
    </div>
    <div class="summary">
      
      
      
      
      &Uopf;se a fenced Div with the class fullwidth, e.g.,
::: {.fullwidth} Any Markdown content or code blocks here. ```{r, echo=FALSE, fig.dim=c(14, 4)} plot(sunspots) ``` :::  Figure 1: Monthly mean relative sunspot numbers from 1749 to 1983. Collected at Swiss Federal Observatory, Zurich until 1960, then Tokyo Astronomical Observatory. [&hellip;] If you add the class fullscroll to the Div, the figure will be scrollable, e.g.,
::: {.fullwidth .fullscroll} :::  Figure 2: This is a super wide figure that you need to scroll to view it fully. [&hellip;] You can embed any elements (typically figures) that span into the page margin. Use a fenced Div with the class embed-left or embed-right, e.g.,
[&hellip;] Figure 3: This is a figure embedded on the right. [&hellip;] ::: {.embed-right} Here is a special figure. ```{r} plot(cars) ``` ::: If you want to embed a table, that will be okay, too. Really, you can embed anything.
::: {.embed-left} ```{r, echo=FALSE} knitr::kable(head(mtcars), caption = &hellip;
      <a href="/post/2020/11/10/r-markdown-demo/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2020/01/07/predictdb-transcriptome-prediction-model-repository/">PredictDB: Transcriptome Prediction Model Repository</a></h1>
    <div>
      <span>2020-01-07</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Hopf;ere you can find transcriptome prediction models for the PrediXcan family of methods: S-PrediXcan, MultiXcan and S-MultiXcan. .db files are prediction models, usable by all methods. .txt.gz files are compilations of LD reference for summary-based methods (S- prefix). S-PrediXcan is meant to use the single-tissue LD reference files (&ldquo;covariances&rdquo;) appropriate to each model. S-MultiXcan uses single-tissue prediction models and a cross-tissue LD reference.
[&hellip;] We have produced different families of prediction models for sQTL and eQTL, using several prediction strategies, on GTEx v8 release data.
We recommend MASHR-based models below. Elastic Net-based are a safe, robust alternative with decreased power.
[&hellip;] Expression and splicing prediction models with LD reference data are available in this Zenodo repository.
Files:
[&hellip;] Warning: these models are based on fine-mapped variants that may occasionally be absent in a tipical GWAS, and frequently absent in &hellip;
      <a href="/post/2020/01/07/predictdb-transcriptome-prediction-model-repository/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2019/12/11/gtex-v8-model-release/">GTEx V8 Model Release</a></h1>
    <div>
      <span>2019-12-11</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Wopf;e have recently published a new set of prediction models trained on GTEx v8 data (as part of efforts detailed in this preprint. We have overhauled the model construction, incorporating posterior inclusion probabilities and global patterns of tissue sharing, while also benefiting from larger sample sizes. We cover both expression and alternate splicing mechanisms. We are very excited about these new models and the potential for new discoveries. However, these models require additional GWAS preprocessing in some public GWAS studies, which we describe here.
We are happy to announce the user-friendly tutorial and detailed documentation.
Our new recommended models use fine-mapped variants (as computed by DAP-G). These variants have a high probability of being causal for QTL. The model effect sizes are computed leveraging cross-tissue patterns with MASHR. These new models are parsimonious, efficient, available for more genes, and have many benefits like improved rate of colocalized &hellip;
      <a href="/post/2019/12/11/gtex-v8-model-release/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2018/04/13/qqplot-calibration-rare-variants/">Qqplot Calibration Rare Variants</a></h1>
    <div>
      <span>2018-04-13</span>
      
    </div>
    <div class="summary">
      
      
      
      
      [In this report] (https://s3.amazonaws.com/imlab-open/Webdata/Files/2018/qqplot-calibration.pdf) I calculate the lower bounds of p-values when using very rare variants, for which minor allele counts are in the single digits. This report was prepared back in 2012 for the T2D-GENES consortium that had just generated 10K whole exome sequenced data.

      <a href="/post/2018/04/13/qqplot-calibration-rare-variants/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2018/01/24/how-to-query-our-gene2pheno-database-directly/">How to query our gene2pheno database directly</a></h1>
    <div>
      <span>2018-01-24</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Wopf;e have opened direct access to the gene2pheno database, where we are hosting the PrediXcan results of close to 3000 phenotypes (from public GWAS meta analysis results and UKBiobank results from Ben Neale/HAIL team).
Below are R functions that will allow you access and query the database. These results are based on GTEx V6p models and details of the analysis can be found in our preprint link to preprint in press now in Nature Communications.
Usual disclaimers apply. This is provided for your convenience and with the hope of being useful. We provide no guarantee of accuracy etc&hellip;
[&hellip;] The code below may not be completely self explanatory. We are working on the documentation.
## First run the preliminary command further down ## this will fetch all associations for Schizophrenia data = query.pheno(&quot;pgc_scz&quot;) ## this will fetch all associations for FTO data = query.gene(&quot;FTO&quot;) ## these rather small tables will be handy phenotb = dbGetQuery(db,&quot;select * &hellip;
      <a href="/post/2018/01/24/how-to-query-our-gene2pheno-database-directly/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>







</main>
</div>

  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  <div class="footer hidden">
  
<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepage_index.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>


  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

