<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.27.1" />
  <meta name="author" content="Hae Kyung Im">
  <meta name="description" content="Assistant Professor - Genetic Medicine - ">

  
  
  
    
  
  
    
    
    <link rel="stylesheet" href="/css/highlight.min.css">
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-61894206-7', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  <link rel="alternate" href="http://hakyimlab.org/index.xml" type="application/rss+xml" title="IM-LAB">
  <link rel="feed" href="http://hakyimlab.org/index.xml" type="application/rss+xml" title="IM-LAB">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="http://hakyimlab.org/post/vulnerabilities/">

  

  <title>Vulnerabilities of the &#39;Vulnerabilities of transcriptome wide association studies&#39; argument | IM-LAB</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">IM-LAB</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/#home">
            
            <span>Home</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#people">
            
            <span>People</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#posts">
            
            <span>News</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#publications">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#talks">
            
            <span>Talks</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#projects">
            
            <span>Software</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Vulnerabilities of the &#39;Vulnerabilities of transcriptome wide association studies&#39; argument</h1>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-11-01 15:35:42 -0500 CDT" itemprop="datePublished">
      Wed, Nov 1, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=http%3a%2f%2fhakyimlab.org%2fpost%2fvulnerabilities%2f"
         target="_blank">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Vulnerabilities%20of%20the%20%27Vulnerabilities%20of%20transcriptome%20wide%20association%20studies%27%20argument&amp;url=http%3a%2f%2fhakyimlab.org%2fpost%2fvulnerabilities%2f"
         target="_blank">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=http%3a%2f%2fhakyimlab.org%2fpost%2fvulnerabilities%2f&amp;title=Vulnerabilities%20of%20the%20%27Vulnerabilities%20of%20transcriptome%20wide%20association%20studies%27%20argument"
         target="_blank">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=http%3a%2f%2fhakyimlab.org%2fpost%2fvulnerabilities%2f&amp;title=Vulnerabilities%20of%20the%20%27Vulnerabilities%20of%20transcriptome%20wide%20association%20studies%27%20argument"
         target="_blank">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Vulnerabilities%20of%20the%20%27Vulnerabilities%20of%20transcriptome%20wide%20association%20studies%27%20argument&amp;body=http%3a%2f%2fhakyimlab.org%2fpost%2fvulnerabilities%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>

    <div class="article-style" itemprop="articleBody">
      <p>A strikingly large number of accomplished researchers seem to interpret significance of a test as proof that the hypothesis is true. Thus it is good to remind ourselves that association does not imply causation.</p>

<p>Wainberg et al’s recent preprint “Vulnerabilities of transcriptome wide association studies” describe limitations of TWAS approaches. A thoughtful response was written by Sasha Gusev (<a href="http://sashagusev.github.io/2017-10/twas-vulnerabilities.html" target="_blank">link</a>).</p>

<p>Below are our responses to some of the claims of the paper (Wainberg et al claims are in indented gray text).</p>

<blockquote>
<p>PrediXcan/TWAS associations do not imply causality</p>
</blockquote>

<p>Yes, we have explicitly stated that we do not claim causality but agree that the message needs to be emphasized.</p>

<blockquote>
<p>TWAS is vulnerable to find multiple associated genes because of co-regulation</p>
</blockquote>

<p>Yes, when a GWAS lead SNP is associated with the expression of several local co-regulated genes, TWAS will yield multiple associated genes. In the same way it is challenging to prove causality, it is also challenging to prove that an associated gene has no causal role. A few examples where we may be relatively confident about causality or the opposite cannot be used to dismiss our approach as done in this manuscript.</p>

<p>No one would argue that a TWAS result implicating multiple local genes should be interpreted as meaning that all of those genes are likely to be contributing to the trait. But assuming that a single gene is always at the root of a GWAS or TWAS signal also seems unnecessarily restrictive.</p>

<blockquote>
<p>“TWAS exhibits instability when tissue/expression panel is changed”</p>
</blockquote>

<p>The expectation that an association found in one tissue should also be found in another one seems to be the problem. A better way to describe these results is that when a gene exhibits tissue-specific genetic regulation, we cannot be surprised to see tissue-specific phenotype associations. This is, we would argue, a feature not a bug.</p>

<p>When the regulatory mechanism is highly context specific, it is possible and likely that none of the reference transcriptome dataset has captured it. This is not a limitation of the approach, but rather of the data available for investigation. We cannot build predictive models without training data, and genes expressed in a single cell type that is poorly represented in available tissues will not be well interrogated with the method, which has been widely acknowledged in the original publications as a limitation of the method.</p>

<blockquote>
<p>“False positive rate is high but unquantifiable”</p>
</blockquote>

<p>This sentence is internally contradictory. But more broadly, would Wainberg et al. consider traditional GWAS to have a high false positive rate due to the large number of associations attributable to LD?</p>

<blockquote>
<p>TWAS is simply a weighted burden test</p>
</blockquote>

<p>PrediXcan/TWAS seeks to test the mediating role of gene expression changes on complex traits. Because we are using natural variation, there are many confounders, such as co-regulation and LD-contamination (GWAs and eQTL causal variants are distinct but in LD), which need to be taken into account but the approach is more than a simple weighted burden test.</p>

<p>Our main concern about this paper is that its negative tone dismisses the utility of PrediXcan/TWAS beyond that of a burden test based on vulnerable arguments. Many groups including ourselves are working on devising ways to mitigate the known limitations of PrediXcan/TWAS but a negative message may force many to abandon this route. We welcome the critique as long as it serves as motivation to continue improving the PrediXcan/TWAS framework.</p>

<hr />

<p>Contributed by Hae Kyung Im and Nancy J Cox</p>

    </div>
  </div>

</article>




<div class="container">
  <nav>
  <ul class="pager">
    
    <li class="previous"><a href="http://hakyimlab.org/post/ubk-neale-predixcan-release/"><span
      aria-hidden="true">&larr;</span> We are releasing all PrediXcan associations based on Neale Lab&#39;s UKB rapid GWAS results</a></li>
    

    
  </ul>
</nav>

</div>

<div class="article-container">
  

</div>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2017 Hae Kyung Im &middot; 

      Powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha512-jGsMH83oKe9asCpkOVkBnUrDDTp8wl+adkB2D+//JtlxO4SrLoJdhbOysIFQJloQFD+C4Fl1rMsQZF76JjV0eQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.2/imagesloaded.pkgd.min.js" integrity="sha512-iHzEu7GbSc705hE2skyH6/AlTpOfBmkx7nUqTLGzPYR+C1tRaItbRlJ7hT/D3YQ9SV0fqLKzp4XY9wKulTBGTw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/TweenMax.min.js" integrity="sha512-Z5heTz36xTemt1TbtbfXtTq5lMfYnOkXM2/eWcTTiLU01+Sw4ku1i7vScDc8fWhrP2abz9GQzgKH5NGBLoYlAw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/plugins/ScrollToPlugin.min.js" integrity="sha512-CDeU7pRtkPX6XJtF/gcFWlEwyaX7mcAp5sO3VIu/ylsdR74wEw4wmBpD5yYTrmMAiAboi9thyBUr1vXRPA7t0Q==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

