<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.46" />
  <meta name="author" content="Hae Kyung Im">
  <meta name="description" content="Assistant Professor - Genetic Medicine - ">

  
  
  
    
  
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-61894206-7', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  <link rel="alternate" href="http://hakyimlab.org/post/index.xml" type="application/rss+xml" title="IM-LAB">
  <link rel="feed" href="http://hakyimlab.org/post/index.xml" type="application/rss+xml" title="IM-LAB">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="http://hakyimlab.org/post/">

  

  <title>Posts | IM-LAB</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">IM-LAB</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/#home">
            
            <span>Home</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#people">
            
            <span>People</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#posts">
            
            <span>News</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#publications">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#talks">
            
            <span>Talks</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#projects">
            
            <span>Software</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>





<div class="universal-wrapper">

  <h1>Posts</h1>

  

  
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2019/gtex-v8-model-release/">GTEx V8 Model Release</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2019-12-11 13:31:33 -0600 CST" itemprop="datePublished">
      Wed, Dec 11, 2019
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      We have recently published a new set of prediction models trained on GTEx v8 data (as part of efforts detailed in this preprint. We have overhauled the model construction, incorporating posterior inclusion probabilities and global patterns of tissue sharing, while also benefiting from larger sample sizes. We cover both expression and alternate splicing mechanisms. We are very excited about these new models and the potential for new discoveries. However, these models require additional GWAS preprocessing in some public GWAS studies, which we describe here.
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/qqplot-calibration-rare-variants/">Qqplot Calibration Rare Variants</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-13 14:22:36 -0500 CDT" itemprop="datePublished">
      Fri, Apr 13, 2018
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      <p><a href="https://s3.amazonaws.com/imlab-open/Webdata/Files/2018/qqplot-calibration.pdf" target="_blank">In this report</a> I calculate the lower bounds of p-values when using very rare variants, for which minor allele counts are in the single digits.
This report was prepared back in 2012 for the T2D-GENES consortium that had just generated 10K whole exome sequenced data.</p>

      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2018/open-access-gene2pheno-dbs/">How to query our gene2pheno database directly</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-01-24 10:43:10 -0600 CST" itemprop="datePublished">
      Wed, Jan 24, 2018
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      We have opened direct access to the gene2pheno database, where we are hosting the PrediXcan results of close to 3000 phenotypes (from public GWAS meta analysis results and UKBiobank results from Ben Neale/HAIL team).
Below are R functions that will allow you access and query the database. These results are based on GTEx V6p models and details of the analysis can be found in our preprint link to preprint in press now in Nature Communications.
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/v7-v6p-analysis/">GTEx V7 Prediction Model Release Announcement</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-01-08 00:00:00 -0600 CST" itemprop="datePublished">
      Mon, Jan 8, 2018
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      We are releasing prediction models trained on GTEx Version 7 data. Download from here. We have updated our processing pipeline, and restricted to individuals of European ancestry to obtain more reliable LD data. This reduces false positive associations in the Summary Version of PrediXcan. Because of this choice, the gain in sample size relative to V6p is modest (ranging from -18 to 89), with whole blood, LCLs and fibroblasts experiencing reduced sample size.
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/scott-farewell/">Bidding Farewell to Scott</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-11-30 13:52:17 -0600 CST" itemprop="datePublished">
      Thu, Nov 30, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      <p>After two years in the Lab, Scott has decided to join the well paid workforce. Scott is a wonderful colleague to all of us and has made important contributions to our team. Thank you and best of lucks, Scott üçÄüçÄüçÄ</p>

<p><img src="https://s3.amazonaws.com/imlab-open/Webdata/Images/2017/scott-farewell.jpg" alt="" /></p>

<p>Also thank you, Wenndy, for organizing and buying the present for Scott.</p>

      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/limitations-predixcan/">Limitations of PrediXcan association results</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-11-07 15:24:42 -0500 -0500" itemprop="datePublished">
      Tue, Nov 7, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      Keep in mind that significant associations shown here do not imply causality. That said, given that PrediXcan is seeking to test the role of gene expression variation on traits and we and others have shown that significant PrediXcan genes are enriched in causal genes, these results should be useful to delve into the mechanisms underlying gene to phenotype associations.
False positives can arise because of several factors
 LD contamination  By computing the probability of LD contamination, we try to reduce false positives due to LD rather than genuine colocalization of trait and expression causal variants.
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/vulnerabilities/">Vulnerabilities of the &#39;Vulnerabilities of transcriptome wide association studies&#39; argument</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-11-07 13:24:42 -0500 -0500" itemprop="datePublished">
      Tue, Nov 7, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      PrediXcan and other transcriptome wide association study (TWAS) methods discover and prioritize genes based on a functional mechanism &ndash;regulation of gene expression. We agree that we have to temper over-enthusiasm, but Wainberg et al&rsquo;s paper could represent a backlash to the enthusiasm that the community has for this approach, which we believe is well placed.
Below are our responses to some of the statements of the paper.
 PrediXcan/TWAS associations do not imply causality
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/robustnes-across-populations/">Transcriptome prediction models are robust across populations</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-11-01 17:19:09 -0500 CDT" itemprop="datePublished">
      Wed, Nov 1, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      Most GWAS and eQTL studies have been performed in European samples. So how well do models trained in Europeans translate to other populations? Segal et al have shown that predictions of gene expression levels are robust across populations (link)
The following figure shows the p-value of the correlation between predicted and observed expression levels in European and African samples from the 1000 Genomes set (GEUVADIS RNA-seq) using model trained in GTEx with majority European individuals.
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/ubk-neale-predixcan-release/">We are releasing all PrediXcan associations based on Neale Lab&#39;s UKB rapid GWAS results</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-10-15 15:38:31 -0400 -0400" itemprop="datePublished">
      Sun, Oct 15, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      Update (11/1/2017)
 Gene level results &ldquo;meta analyzed&rdquo; across tissues now available.
 Shinyapp to directly query by gene and phenotype is here but can be slow. Stay tuned for faster version coming soon.
  As many of you may know, Neale Lab made a big splash in the GWAS community by releasing the summary results of 2400+ phenotypes from the UK Biobank. Following their lead on open science and open data sharing, we are releasing the bulk runs of PrediXcan association based on Neale Lab&rsquo;s UKB results and 44 tissues models from GTEx V6p release.
      
    </p></div>
  </div>
  
  <div>
    
    <h2><a href="http://hakyimlab.org/post/2017/nih-cloud-credit-award/">ImLab was awarded NIH Cloud Credit</a></h2>
    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2017-10-09 10:20:18 -0500 CDT" itemprop="datePublished">
      Mon, Oct 9, 2017
    </time>
  </span>

  
  
  
  

  
  
  
  

  

</div>

    <div class="article-style"><p>
      
      <p>We are delighted to be awarded NIH Cloud Credit that will help us fund our cloud-based web applications and databases and broadly share our tools and resources.</p>

<p>Thank you, Jiamao, for pushing this through üëçüòÉ</p>

<p>Information about Cloud Credits <a href="https://commonfund.nih.gov/bd2k/cloudcredits" target="_blank">here</a></p>

      
    </p></div>
  </div>
  

  
<nav>
  <ul class="pager">
    
    
    <li><a href="/post/page/2/">Next &raquo;</a></li>
    
  </ul>
</nav>



</div>
<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2017 Hae Kyung Im &middot; 

      Powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic
      theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha512-jGsMH83oKe9asCpkOVkBnUrDDTp8wl+adkB2D+//JtlxO4SrLoJdhbOysIFQJloQFD+C4Fl1rMsQZF76JjV0eQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.2/imagesloaded.pkgd.min.js" integrity="sha512-iHzEu7GbSc705hE2skyH6/AlTpOfBmkx7nUqTLGzPYR+C1tRaItbRlJ7hT/D3YQ9SV0fqLKzp4XY9wKulTBGTw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/TweenMax.min.js" integrity="sha512-Z5heTz36xTemt1TbtbfXtTq5lMfYnOkXM2/eWcTTiLU01+Sw4ku1i7vScDc8fWhrP2abz9GQzgKH5NGBLoYlAw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/plugins/ScrollToPlugin.min.js" integrity="sha512-CDeU7pRtkPX6XJtF/gcFWlEwyaX7mcAp5sO3VIu/ylsdR74wEw4wmBpD5yYTrmMAiAboi9thyBUr1vXRPA7t0Q==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    
    <script src="/js/hugo-academic.js"></script>
    

    
    

    
    

  </body>
</html>

