<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> Posts | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Posts"/>
<meta name="twitter:description" content="ImLab website, built with Hugo and blogdown with the hugo-prose theme.
"/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/">Home</a>
      </li>
      
      <li>
        <a href="/project/">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container list">
<main>

<h1>Posts</h1>



<section class="article-list">
<div><p>Here are some recent news.</p>
</div>
</section>










  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2020/12/17/genomic-data-scientist-opening/">Genomic Data Scientist Opening</a></h1>
    <div>
      <span>2020-12-17</span>
      <span>Haky Im</span>
    </div>
    <div class="summary">
      
      
      
      
      [&hellip;] We are looking to hire a life-long learner interested in developing statistical and computational tools to sift through large amounts of data with the ultimate goal of making discoveries that can make a difference in the health of people. Methods and resources created in our lab are being used by researchers around the globe. We continue innovating and providing user-friendly and statistically efficient tools to get the most out of the ever-increasing amount of data. We partner with large consortia and biomedical researchers to find the most pressing questions in the field and apply our statistical expertise to develop reliable and efficient approaches to answer them.

      <a href="/post/2020/12/17/genomic-data-scientist-opening/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2020/11/10/r-markdown-demo/">R Markdown Demo</a></h1>
    <div>
      <span>2020-11-10</span>
      <span>Yihui Xie</span>
    </div>
    <div class="summary">
      
      
      
      
      &Uopf;se a fenced Div with the class fullwidth, e.g.,
::: {.fullwidth} Any Markdown content or code blocks here. ```{r, echo=FALSE, fig.dim=c(14, 4)} plot(sunspots) ``` :::  Figure 1: Monthly mean relative sunspot numbers from 1749 to 1983. Collected at Swiss Federal Observatory, Zurich until 1960, then Tokyo Astronomical Observatory. [&hellip;] If you add the class fullscroll to the Div, the figure will be scrollable, e.g.,
::: {.fullwidth .fullscroll} :::  Figure 2: This is a super wide figure that you need to scroll to view it fully. [&hellip;] You can embed any elements (typically figures) that span into the page margin. Use a fenced Div with the class embed-left or embed-right, e.g.,
[&hellip;] Figure 3: This is a figure embedded on the right. [&hellip;] ::: {.embed-right} Here is a special figure. ```{r} plot(cars) ``` ::: If you want to embed a table, that will be okay, too. Really, you can embed anything.
::: {.embed-left} ```{r, echo=FALSE} knitr::kable(head(mtcars), caption = &hellip;
      <a href="/post/2020/11/10/r-markdown-demo/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2020/01/07/predictdb-transcriptome-prediction-model-repository/">PredictDB: Transcriptome Prediction Model Repository</a></h1>
    <div>
      <span>2020-01-07</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Hopf;ere you can find transcriptome prediction models for the PrediXcan family of methods: S-PrediXcan, MultiXcan and S-MultiXcan. .db files are prediction models, usable by all methods. .txt.gz files are compilations of LD reference for summary-based methods (S- prefix). S-PrediXcan is meant to use the single-tissue LD reference files (&ldquo;covariances&rdquo;) appropriate to each model. S-MultiXcan uses single-tissue prediction models and a cross-tissue LD reference.
[&hellip;] We have produced different families of prediction models for sQTL and eQTL, using several prediction strategies, on GTEx v8 release data.
We recommend MASHR-based models below. Elastic Net-based are a safe, robust alternative with decreased power.
[&hellip;] Expression and splicing prediction models with LD reference data are available in this Zenodo repository.
Files:
[&hellip;] Warning: these models are based on fine-mapped variants that may occasionally be absent in a tipical GWAS, and frequently absent in &hellip;
      <a href="/post/2020/01/07/predictdb-transcriptome-prediction-model-repository/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2019/12/11/gtex-v8-model-release/">GTEx V8 Model Release</a></h1>
    <div>
      <span>2019-12-11</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Wopf;e have recently published a new set of prediction models trained on GTEx v8 data (as part of efforts detailed in this preprint. We have overhauled the model construction, incorporating posterior inclusion probabilities and global patterns of tissue sharing, while also benefiting from larger sample sizes. We cover both expression and alternate splicing mechanisms. We are very excited about these new models and the potential for new discoveries. However, these models require additional GWAS preprocessing in some public GWAS studies, which we describe here.
We are happy to announce the user-friendly tutorial and detailed documentation.
Our new recommended models use fine-mapped variants (as computed by DAP-G). These variants have a high probability of being causal for QTL. The model effect sizes are computed leveraging cross-tissue patterns with MASHR. These new models are parsimonious, efficient, available for more genes, and have many benefits like improved rate of colocalized &hellip;
      <a href="/post/2019/12/11/gtex-v8-model-release/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2018/04/13/qqplot-calibration-rare-variants/">Qqplot Calibration Rare Variants</a></h1>
    <div>
      <span>2018-04-13</span>
      
    </div>
    <div class="summary">
      
      
      
      
      [In this report] (https://s3.amazonaws.com/imlab-open/Webdata/Files/2018/qqplot-calibration.pdf) I calculate the lower bounds of p-values when using very rare variants, for which minor allele counts are in the single digits. This report was prepared back in 2012 for the T2D-GENES consortium that had just generated 10K whole exome sequenced data.

      <a href="/post/2018/04/13/qqplot-calibration-rare-variants/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2018/01/24/how-to-query-our-gene2pheno-database-directly/">How to query our gene2pheno database directly</a></h1>
    <div>
      <span>2018-01-24</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Wopf;e have opened direct access to the gene2pheno database, where we are hosting the PrediXcan results of close to 3000 phenotypes (from public GWAS meta analysis results and UKBiobank results from Ben Neale/HAIL team).
Below are R functions that will allow you access and query the database. These results are based on GTEx V6p models and details of the analysis can be found in our preprint link to preprint in press now in Nature Communications.
Usual disclaimers apply. This is provided for your convenience and with the hope of being useful. We provide no guarantee of accuracy etc&hellip;
[&hellip;] The code below may not be completely self explanatory. We are working on the documentation.
## First run the preliminary command further down ## this will fetch all associations for Schizophrenia data = query.pheno(&quot;pgc_scz&quot;) ## this will fetch all associations for FTO data = query.gene(&quot;FTO&quot;) ## these rather small tables will be handy phenotb = dbGetQuery(db,&quot;select * &hellip;
      <a href="/post/2018/01/24/how-to-query-our-gene2pheno-database-directly/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2018/01/08/gtex-v7-prediction-model-release-announcement/">GTEx V7 Prediction Model Release Announcement</a></h1>
    <div>
      <span>2018-01-08</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Wopf;e are releasing prediction models trained on GTEx Version 7 data. Download from here. We have updated our processing pipeline, and restricted to individuals of European ancestry to obtain more reliable LD data. This reduces false positive associations in the Summary Version of PrediXcan. Because of this choice, the gain in sample size relative to V6p is modest (ranging from -18 to 89), with whole blood, LCLs and fibroblasts experiencing reduced sample size. We developed new criteria to assess model performance. We have also decided to include prediction models for both pseudogenes and lincRNAs.
While preparing GTEx V7 prediction models, we identified a few issues in the way prediction performance was estimated in the previous release (2016-09-08 release). In aggregate, these caused the prediction model performance to be overestimated. Reassuringly, predicted expression levels and the downstream associations with phenotypes remain mostly unchanged, even though prediction weights vary &hellip;
      <a href="/post/2018/01/08/gtex-v7-prediction-model-release-announcement/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/11/30/bidding-farewell-to-scott/">Bidding Farewell to Scott</a></h1>
    <div>
      <span>2017-11-30</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Aopf;fter two years in the Lab, Scott has decided to join the well paid workforce. Scott is a wonderful colleague to all of us and has made important contributions to our team. Thank you and best of lucks, Scott :four_leaf_clover::four_leaf_clover::four_leaf_clover:
Also thank you, Wenndy, for organizing and buying the present for Scott.

      <a href="/post/2017/11/30/bidding-farewell-to-scott/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/11/07/limitations-of-predixcan-association-results/">Limitations of PrediXcan association results</a></h1>
    <div>
      <span>2017-11-07</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Kopf;eep in mind that significant associations shown here do not imply causality. That said, given that PrediXcan is seeking to test the role of gene expression variation on traits and we and others have shown that significant PrediXcan genes are enriched in causal genes, these results should be useful to delve into the mechanisms underlying gene to phenotype associations.
False positives can arise because of several factors
[&hellip;] By computing the probability of LD contamination, we try to reduce false positives due to LD rather than genuine colocalization of trait and expression causal variants. Filtering out genes with high probability of independent signals (P3&gt;0.5) would reduce the number of hits due to LD contamination.
[&hellip;] Many loci show regulation of multiple genes. PrediXcan is not able to distinguish between truly causal and co-regulation but non causal genes.
[&hellip;] Misspecified LD can lead to false positives (for example if LD in the study population is much &hellip;
      <a href="/post/2017/11/07/limitations-of-predixcan-association-results/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/11/07/vulnerabilities-of-the-vulnerabilities-of-transcriptome-wide-association-studies-argument/">Vulnerabilities of the &#39;Vulnerabilities of transcriptome wide association studies&#39; argument</a></h1>
    <div>
      <span>2017-11-07</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Popf;rediXcan and other transcriptome wide association study (TWAS) methods discover and prioritize genes based on a functional mechanism &ndash;regulation of gene expression. We agree that we have to temper over-enthusiasm, but Wainberg et al&rsquo;s paper could represent a backlash to the enthusiasm that the community has for this approach, which we believe is well placed.
Below are our responses to some of the statements of the paper.
[&hellip;] PrediXcan/TWAS associations do not imply causality
[&hellip;] Yes, we have explicitly stated that we do not claim causality but agree that the message needs to be emphasized. However, the over-interpretation of TWAS results as causal (which itself seems overstated by Wainberg et al), is precisely because of the exciting aspect of TWAS as association testing ‘through a lens’ of a particular functional mechanism that we believe is very important, namely, regulation of gene expression.
[&hellip;] TWAS is vulnerable to find multiple associated genes &hellip;
      <a href="/post/2017/11/07/vulnerabilities-of-the-vulnerabilities-of-transcriptome-wide-association-studies-argument/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>




<ul class="pagination">
  <li class="page-item">
    <a href="/post/" class="page-link" aria-label="First"><span aria-hidden="true">&laquo;&laquo;</span></a>
  </li>
  <li class="page-item disabled">
    <a  class="page-link" aria-label="Previous"><span aria-hidden="true">&laquo;</span></a>
  </li>
  <li class="page-item active">
    <a class="page-link" href="/post/">1</a>
  </li>
  <li class="page-item">
    <a class="page-link" href="/post/page/2/">2</a>
  </li>
  <li class="page-item">
    <a class="page-link" href="/post/page/3/">3</a>
  </li>
  <li class="page-item">
    <a href="/post/page/2/" class="page-link" aria-label="Next"><span aria-hidden="true">&raquo;</span></a>
  </li>
  <li class="page-item">
    <a href="/post/page/3/" class="page-link" aria-label="Last"><span aria-hidden="true">&raquo;&raquo;</span></a>
  </li>
</ul>






<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//hakyimlab.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    var t = d.getElementById('disqus_thread');
    var b = false, l = function(scroll) {
      if (b) return;
      (d.head || d.body).appendChild(s); b = true;
      if (scroll) t.scrollIntoView();
    }
    s.onerror = function(e) {
      if (sessionStorage.getItem('failure-note')) return;
      t.innerText = 'Sorry, but you cannot make comments because Disqus failed to load for some reason. It is known to be blocked in China. If you are sure it is not blocked in your region, please refresh the page. 中国大陆地区读者需要翻墙才能发表评论。';
      t.style.border = '1px dashed';
      t.style.padding = '.5em';
      t.style.background = 'lightyellow';
      sessionStorage.setItem('failure-note', true);
    };
    
    if (location.hash.match(/^#comment-[0-9]+$/)) return l(true);
    var c = function() {
      if (b) return;
      var rect = t.getBoundingClientRect();
      if (rect.top < window.innerHeight && rect.bottom >= 0) l();
    };
    window.addEventListener('load', c);
    d.addEventListener('scroll', c);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>


</main>
</div>

  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  <div class="footer hidden">
  
<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepagepost/_index.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>


  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

