<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> Posts | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Posts"/>
<meta name="twitter:description" content="ImLab website, built with Hugo and blogdown with the hugo-prose theme.
"/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/">Home</a>
      </li>
      
      <li>
        <a href="/project/">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container list">
<main>

<h1>Posts</h1>



<section class="article-list">
<div><p>Here are some recent news.</p>
</div>
</section>










  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/11/01/transcriptome-prediction-models-are-robust-across-populations/">Transcriptome prediction models are robust across populations</a></h1>
    <div>
      <span>2017-11-01</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Mopf;ost GWAS and eQTL studies have been performed in European samples. So how well do models trained in Europeans translate to other populations? Segal et al have shown that predictions of gene expression levels are robust across populations (link)
The following figure shows the p-value of the correlation between predicted and observed expression levels in European and African samples from the 1000 Genomes set (GEUVADIS RNA-seq) using model trained in GTEx with majority European individuals. As expected performance decreases (blue is more significant than red) when we apply to African samples but the correlation is still quite significant. 
      <a href="/post/2017/11/01/transcriptome-prediction-models-are-robust-across-populations/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/10/15/we-are-releasing-all-predixcan-associations-based-on-neale-labs-ukb-rapid-gwas-results/">We are releasing all PrediXcan associations based on Neale Lab&#39;s UKB rapid GWAS results</a></h1>
    <div>
      <span>2017-10-15</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Uopf;pdate (11/1/2017)
[&hellip;] Gene level results &ldquo;meta analyzed&rdquo; across tissues now available.
[&hellip;] Shinyapp to directly query by gene and phenotype is here but can be slow. Stay tuned for faster version coming soon.
[&hellip;] As many of you may know, Neale Lab made a big splash in the GWAS community by releasing the summary results of 2400+ phenotypes from the UK Biobank. Following their lead on open science and open data sharing, we are releasing the bulk runs of PrediXcan association based on Neale Lab&rsquo;s UKB results and 44 tissues models from GTEx V6p release.
All PrediXcan associations can be downloaded from here
We are also generating a &ldquo;meta analyzed&rdquo; across tissues version, which we find much more easily interpretable. They will be uploaded within the next few days.
All this is work in progress, so feedback is greatly appreciated! We are trying to flag some suspicous results (likely false positives) but so please let us know if you find them. &hellip;
      <a href="/post/2017/10/15/we-are-releasing-all-predixcan-associations-based-on-neale-labs-ukb-rapid-gwas-results/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/10/09/imlab-was-awarded-nih-cloud-credit/">ImLab was awarded NIH Cloud Credit</a></h1>
    <div>
      <span>2017-10-09</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Wopf;e are delighted to be awarded NIH Cloud Credit that will help us fund our cloud-based web applications and databases and broadly share our tools and resources.
Thank you, Jiamao, for pushing this through :thumbsup::smiley:
Information about Cloud Credits here

      <a href="/post/2017/10/09/imlab-was-awarded-nih-cloud-credit/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/09/28/u4c-challenge-winners/">U4C Challenge Winners</a></h1>
    <div>
      <span>2017-09-28</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Topf;wo out of 4 winners used PrediXcan :thumbsup::thumbsup:
Winners of &ldquo;Up For A Challenge (U4C): Stimulating innovation in breast cancer genetic epidemiology&rdquo; papers published on line link here.
Here is our contribution in collaboration with Guimin Gao, Brandon Pierce, Funmi Olopade, and Dezheng Huo link to paper here

      <a href="/post/2017/09/28/u4c-challenge-winners/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/08/14/s-predixcan-metaxcan-in-drug-repositioning/">S-PrediXcan (MetaXcan) in drug repositioning</a></h1>
    <div>
      <span>2017-08-14</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Sopf;o et al use S-PrediXcan to find psychiatric drug repositioning candidates.

      <a href="/post/2017/08/14/s-predixcan-metaxcan-in-drug-repositioning/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/08/04/lab-summer-photo/">lab summer photo</a></h1>
    <div>
      <span>2017-08-04</span>
      
    </div>
    <div class="summary">
      
      
      
      
      
      <a href="/post/2017/08/04/lab-summer-photo/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/07/27/lab-photo-summer-2017/">Lab Photo Summer 2017</a></h1>
    <div>
      <span>2017-07-27</span>
      
    </div>
    <div class="summary">
      
      
      
      
      
      <a href="/post/2017/07/27/lab-photo-summer-2017/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2017/06/27/dinner-with-nancy-at-the-gtex-meeting/">Dinner with Nancy at the GTEx meeting</a></h1>
    <div>
      <span>2017-06-27</span>
      <span>Haky</span>
    </div>
    <div class="summary">
      
      
      
      
      &Aopf;fter the last formal GTEx analysis working group&rsquo;s internal meeting at Rockville, MD, the Im Lab went for dinner with Nancy Cox, Barbara Stranger, and Eric Gamazon. From left to right: Rodrigo, Milton, Jiamao, Barbara, Nancy, Eric, Haky, Scott, and Alvaro. Great French dishes with Asian influence.

      <a href="/post/2017/06/27/dinner-with-nancy-at-the-gtex-meeting/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    
    <div class="categories">
      <a href="/categories/r/">R</a>
    </div>
    
    
    <h1><a href="/post/2015/07/23/hello-r-markdown/">Hello R Markdown</a></h1>
    <div>
      <span>2015-07-23</span>
      <span>Frida Gomam</span>
    </div>
    <div class="summary">
      
      
      
      
      &Topf;his is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see http://rmarkdown.rstudio.com.
You can embed an R code chunk like this:
summary(cars) ## speed dist ## Min. : 4.0 Min. : 2.00 ## 1st Qu.:12.0 1st Qu.: 26.00 ## Median :15.0 Median : 36.00 ## Mean :15.4 Mean : 42.98 ## 3rd Qu.:19.0 3rd Qu.: 56.00 ## Max. :25.0 Max. :120.00 fit &lt;- lm(dist ~ speed, data = cars) fit ## ## Call: ## lm(formula = dist ~ speed, data = cars) ## ## Coefficients: ## (Intercept) speed ## -17.579 3.932  Including Plots You can also embed plots. See Figure 1 for example:
par(mar = c(0, 1, 0, 1)) pie( c(280, 60, 20), c(&#39;Sky&#39;, &#39;Sunny side of pyramid&#39;, &#39;Shady side of pyramid&#39;), col = c(&#39;#0292D8&#39;, &#39;#F7EA39&#39;, &#39;#C4B632&#39;), init.angle = -50, border = NA )  Figure 1: A fancy pie chart. 
      <a href="/post/2015/07/23/hello-r-markdown/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/post/2000/01/01/comment-policy/">Comment Policy</a></h1>
    <div>
      <span>2000-01-01</span>
      
    </div>
    <div class="summary">
      
      
      
      
      &Aopf;void:
[&hellip;] Comments that we find to be hateful, inflammatory, or harassing may be removed. If you don’t have something nice to say about another user, don&rsquo;t say it. Treat others the way you’d like to be treated.
[&hellip;] There are a lot of discussions that happen every day on Disqus. Before joining in a discussion, browse through some of the most recent and active discussions happening in the community, especially if you’re new there.
If you are not sure your post adds to the conversation, think over what you want to say and try again later.
[&hellip;] Help make moderators’ lives easier by taking a moment to ensure that what you’re about to post is in the right place. That means: don’t post off-topic comments or discussions: check the Explore page to find the appropriate channel for your topic don’t cross-post the same thing multiple times review any specific posting guidelines for the community: some communities such as a movies community may have specific rules &hellip;
      <a href="/post/2000/01/01/comment-policy/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>




<ul class="pagination">
  <li class="page-item">
    <a href="/post/" class="page-link" aria-label="First"><span aria-hidden="true">&laquo;&laquo;</span></a>
  </li>
  <li class="page-item">
    <a href="/post/" class="page-link" aria-label="Previous"><span aria-hidden="true">&laquo;</span></a>
  </li>
  <li class="page-item">
    <a class="page-link" href="/post/">1</a>
  </li>
  <li class="page-item active">
    <a class="page-link" href="/post/page/2/">2</a>
  </li>
  <li class="page-item">
    <a class="page-link" href="/post/page/3/">3</a>
  </li>
  <li class="page-item">
    <a href="/post/page/3/" class="page-link" aria-label="Next"><span aria-hidden="true">&raquo;</span></a>
  </li>
  <li class="page-item">
    <a href="/post/page/3/" class="page-link" aria-label="Last"><span aria-hidden="true">&raquo;&raquo;</span></a>
  </li>
</ul>






<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//hakyimlab.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    var t = d.getElementById('disqus_thread');
    var b = false, l = function(scroll) {
      if (b) return;
      (d.head || d.body).appendChild(s); b = true;
      if (scroll) t.scrollIntoView();
    }
    s.onerror = function(e) {
      if (sessionStorage.getItem('failure-note')) return;
      t.innerText = 'Sorry, but you cannot make comments because Disqus failed to load for some reason. It is known to be blocked in China. If you are sure it is not blocked in your region, please refresh the page. 中国大陆地区读者需要翻墙才能发表评论。';
      t.style.border = '1px dashed';
      t.style.padding = '.5em';
      t.style.background = 'lightyellow';
      sessionStorage.setItem('failure-note', true);
    };
    
    if (location.hash.match(/^#comment-[0-9]+$/)) return l(true);
    var c = function() {
      if (b) return;
      var rect = t.getBoundingClientRect();
      if (rect.top < window.innerHeight && rect.bottom >= 0) l();
    };
    window.addEventListener('load', c);
    d.addEventListener('scroll', c);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>


</main>
</div>

  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  <div class="footer hidden">
  
<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepagepost/_index.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>


  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

