<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> PredictDB: Transcriptome Prediction Model Repository | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="PredictDB: Transcriptome Prediction Model Repository"/>
<meta name="twitter:description" content="Predictdb Here you can find transcriptome prediction models for the PrediXcan family of methods: S-PrediXcan, MultiXcan and S-MultiXcan. .db files are prediction models, usable by all methods. .txt.gz files are compilations of LD reference for summary-based methods (S- prefix). S-PrediXcan is meant to use the single-tissue LD reference files (&ldquo;covariances&rdquo;) appropriate to each model. S-MultiXcan uses single-tissue prediction models and a cross-tissue LD reference.
GTEx v8 models on eQTL and sQTL We have produced different families of prediction models for sQTL and eQTL, using several prediction strategies, on GTEx v8 release data."/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/">Home</a>
      </li>
      
      <li>
        <a href="/project/">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container single">
<main>

<div class="article-meta">
<h1><span class="title">PredictDB: Transcriptome Prediction Model Repository</span></h1>

<h3 class="date">2020-01-07</h3>
<p class="terms">
  
  
  
  
  
</p>
</div>

<div class="article">
<h1 id="predictdb">Predictdb</h1>
<p>Here you can find transcriptome prediction models for the PrediXcan family of methods: S-PrediXcan, MultiXcan and S-MultiXcan.
<code>.db</code> files are prediction models, usable by all methods.
<code>.txt.gz</code> files are compilations of LD reference for summary-based methods (<code>S-</code> prefix).
S-PrediXcan is meant to use the single-tissue LD reference files (&ldquo;covariances&rdquo;) appropriate to each model.
S-MultiXcan uses single-tissue prediction models and a cross-tissue LD reference.</p>
<h1 id="gtex-v8-models-on-eqtl-and-sqtl">GTEx v8 models on eQTL and sQTL</h1>
<p>We have produced different families of prediction models for sQTL and eQTL,
using several prediction strategies, on GTEx v8 release data.</p>
<p>We recommend MASHR-based models below.
Elastic Net-based are a safe, robust alternative with decreased power.</p>
<h2 id="mashr-based-models">MASHR-based models</h2>
<p>Expression and splicing prediction models with LD reference data are available in this
<a href="https://zenodo.org/record/3518299">Zenodo repository</a>.</p>
<p>Files:</p>
<ul>
<li><a href="https://zenodo.org/record/3518299/files/mashr_eqtl.tar?download=1">mashr_eqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s support on expression</li>
<li><a href="https://zenodo.org/record/3518299/files/mashr_sqtl.tar?download=1">mashr_sqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s support on splicing</li>
<li><a href="https://zenodo.org/record/3518299/files/gtex_v8_expression_mashr_snp_smultixcan_covariance.txt.gz?download=1">gtex_v8_expression_mashr_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan expression&rsquo;s LD reference</li>
<li><a href="https://zenodo.org/record/3518299/files/gtex_v8_splicing_mashr_snp_smultixcan_covariance.txt.gz?download=1">gtex_v8_splicing_mashr_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan splicing&rsquo;s LD reference</li>
</ul>
<p><strong>Warning</strong>: these models are based on fine-mapped variants that may occasionally be absent
in a tipical GWAS, and frequently absent in older GWAS.
We have tools to address this, presented
<a href="https://github.com/hakyimlab/MetaXcan/wiki/Best-practices-for-integrating-GWAS-and-GTEX-v8-transcriptome-prediction-models">here</a>.
A tutorial is available <a href="https://github.com/hakyimlab/MetaXcan/wiki/Tutorial:-GTEx-v8-MASH-models-integration-with-a-Coronary-Artery-Disease-GWAS">here</a>.</p>
<p><strong>Acknowledging these models</strong>: If you use these models in your research, please cite:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/814350v1">&ldquo;Widespread dose-dependent effects of RNA expression and splicing on complex diseases and traits&rdquo;</a>, Barbeira et al, 2019, preprint</li>
<li><a href="https://www.nature.com/articles/ng.3367">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics</li>
<li><a href="https://www.nature.com/articles/s41467-018-03621-1">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</li>
</ul>
<p>If you use S-MultiXcan, we ask you to cite:</p>
<ul>
<li><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</li>
</ul>
<h2 id="elastic-net">Elastic Net</h2>
<p>Expression and splicing prediction models with LD references data are available in this
<a href="https://zenodo.org/record/3519321#.XfqJz9l7m90">Zenodo repository</a>.</p>
<p>Files:</p>
<ul>
<li><a href="https://zenodo.org/record/3519321/files/elastic_net_eqtl.tar?download=1">elastic_net_eqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s support on expression</li>
<li><a href="https://zenodo.org/record/3519321/files/elastic_net_sqtl.tar?download=1">elastic_net_sqtl.tar</a>: PrediXcan&rsquo;s and S-PrediXcan&rsquo;s on splicing</li>
<li><a href="https://zenodo.org/record/3519321/files/gtex_v8_expression_elastic_net_snp_smultixcan_covariance.txt.gz?download=1">gtex_v8_expression_elastic_net_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan expression&rsquo;s LD reference</li>
<li><a href="https://zenodo.org/record/3519321/files/gtex_v8_splicing_elastic_net_snp_smultixcan_covariance.txt.gz?download=1">gtex_v8_splicing_elastic_net_snp_smultixcan_covariance.txt.gz</a>: S-MultiXcan splicing&rsquo;s LD reference</li>
</ul>
<p><strong>Acknowledging these models</strong> : If you use these models in your research, we ask you to cite:</p>
<ul>
<li><a href="https://www.biorxiv.org/content/10.1101/787903v1">&ldquo;The GTEx Consortium atlas of genetic regulatory effects across human tissues&rdquo;</a>, Aguet et al, 2019, preprint</li>
<li><a href="https://www.nature.com/articles/ng.3367">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics</li>
<li><a href="https://www.nature.com/articles/s41467-018-03621-1">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</li>
</ul>
<p>If you use S-MultiXcan, we ask you to cite:</p>
<ul>
<li><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</li>
</ul>
<h1 id="gtex-v7-expression-models">GTEx v7 Expression models</h1>
<p>Expression prediction models with LD reference data ar available in this
<a href="https://zenodo.org/record/3572799">Zenodo repository</a>.
The underlying algorithm is Elastic Net.</p>
<ul>
<li><a href="https://zenodo.org/record/3572799/files/GTEx-V7_HapMap-2017-11-29.tar.gz?download=1">GTEx-V7_HapMap-2017-11-29.tar.gz</a>: Expression prediction models and LD reference on single tissues.</li>
<li><a href="https://zenodo.org/record/3572799/files/snp_covariance_v7.txt.gz?download=1">snp_covariance_v7.txt.gz</a>: S-MultiXcan&rsquo;s LD compilation</li>
</ul>
<p>Additional support information and details are available in the Zenodo repository.</p>
<p><strong>Acknowledging these models</strong>: If you use these models in your research, we ask you to cite:</p>
<ul>
<li><a href="https://www.nature.com/articles/ng.3367">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics</li>
<li><a href="https://www.nature.com/articles/s41467-018-03621-1">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</li>
</ul>
<p>If you use S-MultiXcan, we ask you to cite:</p>
<ul>
<li><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</li>
</ul>
<h1 id="gtex-v6-expression-models">GTEx v6 Expression models</h1>
<p>Expression prediction models with LD reference data are available in this
<a href="https://zenodo.org/record/3572842#.XfuyLNl7m90">Zenodo repository</a>.</p>
<ul>
<li><a href="https://zenodo.org/record/3572842/files/GTEx-V6p-1KG-2016-11-16.tar.gz?download=1">GTEx-V6p-1KG-2016-11-16.tar.gz</a>: Expression prediction models and LD reference, using variants defined in 1000 Genomes</li>
<li><a href="https://zenodo.org/record/3572842/files/GTEx-V6p-HapMap-2016-09-08.tar.gz?download=1">GTEx-V6p-HapMap-2016-09-08.tar.gz</a>: Expression prediction models and LD reference, using variants defined in Hapmap CEU track</li>
<li><a href="https://zenodo.org/record/3572842/files/snp_covariance_v6p.txt.gz?download=1">snp_covariance_v6p.txt.gz</a>: S-MultiXcan LD reference for HapMap models</li>
</ul>
<p>Additional support information and details are available in the Zenodo repository.</p>
<p><strong>Acknowledging these models</strong>: If you use these models in your research, we ask you to cite:</p>
<ul>
<li><a href="https://www.nature.com/articles/ng.3367">&ldquo;A gene-based association method for mapping traits using reference transcriptome data&rdquo;</a>, Gamazon et al, 2015, Nature Genetics</li>
<li><a href="https://www.nature.com/articles/s41467-018-03621-1">&ldquo;Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics&rdquo;</a>, Barbeira et al, 2018,  Nature Communications</li>
</ul>
<p>If you use S-MultiXcan, we ask you to cite:</p>
<ul>
<li><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007889">&ldquo;Integrating predicted transcriptome from multiple tissues improves association detection&rdquo;</a>, Barbeira et al, 2019, PLOS Genetics</li>
</ul>
<h1 id="models-from-collaborators-and-other-sources">Models from collaborators and other sources:</h1>
<h2 id="mesa-models">MESA models</h2>
<p>Single-tissue expression prediction models with LD reference data are available in this
<a href="https://zenodo.org/record/3572815#.XfqOstl7m90">Zenodo repository</a>.
The underlying algorithm is Elastic Net on MESA multi-ethnic cohort.</p>
<p>These models were presented in <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007586">&ldquo;Genetic architecture of gene expression traits across diverse populations&rdquo;</a>, Mogil et al, 2018, PLOS Genetics.
Please cite if you find these useful.</p>
<h2 id="commonmind-consortium">CommonMind consortium</h2>
<p>Single-tissue expression prediction models with LD reference data are available in this
<a href="https://github.com/laurahuckins/CMC_DLPFC_prediXcan">GitHub repository</a>.
The underlying algorithm is Elastic Net.</p>
<p>These models were presented in <a href="https://www.nature.com/articles/s41588-019-0364-4">&ldquo;Gene expression imputation across multiple brain regions provides insights into schizophrenia risk</a>,
Huckins et al, 2019, Nature Genetics. Please cite if you find these useful.</p>
<h2 id="epixcan-models">EpiXcan Models</h2>
<p>Expression prediction models with LD reference data are available in this <a href="https://zhangw17.u.hpc.mssm.edu/epixcan/about.php#dl">website</a>.
The models were trained on Common Mind Consortium, GTEx, and STARNET consortiums.
The underlying algorithm is Elastic Net, informed by epigenetic data.</p>
<p>These models were presented in <a href="https://www.nature.com/articles/s41467-019-11874-7">&ldquo;Integrative transcriptome imputation reveals tissue-specific and shared biological mechanisms mediating susceptibility to complex traits&rdquo;</a>,
Zhaneg et al, 2019, Nature Communications. Please cite if you find these useful.</p>

</div>
</main>

<section class="appendix">







<div>
  <div class="side side-left"><h3>Reuse</h3></div>
  Text and figures are licensed under <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution CC BY 4.0</a>. The source code is licensed under MIT.
</div>



<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepagepost/2020/predictdb.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>




</section>



<nav class="post-nav">
  <span class="nav-next">&larr; <a href="/post/2019/12/11/gtex-v8-model-release/" title="Next post (older)">GTEx V8 Model Release</a></span>
  &hercon;
  <span class="nav-prev"><a href="/post/2020/11/10/r-markdown-demo/" title="Previous post (newer)">R Markdown Demo</a> &rarr;</span>
</nav>




<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//hakyimlab.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    var t = d.getElementById('disqus_thread');
    var b = false, l = function(scroll) {
      if (b) return;
      (d.head || d.body).appendChild(s); b = true;
      if (scroll) t.scrollIntoView();
    }
    s.onerror = function(e) {
      if (sessionStorage.getItem('failure-note')) return;
      t.innerText = 'Sorry, but you cannot make comments because Disqus failed to load for some reason. It is known to be blocked in China. If you are sure it is not blocked in your region, please refresh the page. 中国大陆地区读者需要翻墙才能发表评论。';
      t.style.border = '1px dashed';
      t.style.padding = '.5em';
      t.style.background = 'lightyellow';
      sessionStorage.setItem('failure-note', true);
    };
    
    if (location.hash.match(/^#comment-[0-9]+$/)) return l(true);
    var c = function() {
      if (b) return;
      var rect = t.getBoundingClientRect();
      if (rect.top < window.innerHeight && rect.bottom >= 0) l();
    };
    window.addEventListener('load', c);
    d.addEventListener('scroll', c);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>


</div>


  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

