<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <title> Research | IM-Lab</title>
    
    
    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="/css/custom.css" />
    <link href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" rel="stylesheet">

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Research"/>
<meta name="twitter:description" content="ImLab website, built with Hugo and blogdown with the hugo-prose theme.
"/>


  </head>

  <body>

    <nav class="menu">
    <ul>
      <li class="left">
        <a href="/"><span>IM-Lab</span></a>
      </li>
      
      <li>
        <a href="/">Home</a>
      </li>
      
      <li>
        <a href="/project/" class="active">Research</a>
      </li>
      
      <li>
        <a href="/resource/">Resource</a>
      </li>
      
      <li>
        <a href="/sitemap/">Sitemap</a>
      </li>
      
      <li id="menu-search">
        <a href="/#">Search</a>
      </li>
      
    </ul>
    </nav>


<div class="container list">
<main>

<h1>Research</h1>



<section class="article-list">
<div><p>The overarching goal of our research program is to develop computationally efficient and statistically sound methods to sift through the vast amounts of genomic and other high dimensional data with the goal of making discoveries that can be translated to improve human health.</p>
<p>To achieve this goal, we build and maintain an analytic infrastructure, i.e. a dry lab. This includes statistical and computational methods, pipelines and workflows, streamlined access to big data, and a research software engineering, bioinformatic and analysis team. This infrastructure constitutes a powerful instrument that gives us a unique vantage point from which discoveries can be made.</p>
<p>The cards below describe some of the highlights of our research.</p>
</div>
</section>










  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/project/other-projects/">Published Research</a></h1>
    <div>
      
      
    </div>
    <div class="summary">
      
      
      
      
      &Gopf;enomic privacy and data sharing are issues of relevance for the whole scientific community. Protecting the privacy of individuals who participate in a study has always been a top priority and it has been widely assumed that publishing summary results did not jeopardize privacy. In 2008, Homer et al found that in the case of genome wide association studies (GWAS), summary results such as allele frequencies for a large number of genetic variants can reveal whether a person participated in a study and the disease status of the individual. These results forced the NIH to withdraw most of the public access to GWAS study results. We were interested in sharing results from quantitative traits such as gene expression phenotypes, which provide critical information on the regulatory role of genetic variants. The question here was whether publishing regression coefficients from GWAS would also allow re-identification. Dr. Im proved mathematically that re-identification based on regression &hellip;
      <a href="/project/other-projects/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/project/published-research/">Published Research</a></h1>
    <div>
      
      
    </div>
    <div class="summary">
      
      
      
      
      &Aopf;lthough investments in genomic studies of complex diseases made possible the discovery of thousands of variants robustly associated with these traits, the translation of these discoveries into actionable targets has been hampered by the lack of a mechanistic understanding on how genome variation relates to phenotype. Moreover, it has been widely shown that a substantial portion of the genetic control of complex traits is exerted through the regulation of gene expression traits, but effective methods to fully harness this mechanism were not available. To address these challenges, we developed a gene-based test —PrediXcan— that directly tests this regulatory mechanism and substantially improves power relative to single variant tests and other gene-based tests. PrediXcan is inherently mechanistic and provides directionality, highlighting its potential utility in identifying novel targets for therapy. PrediXcan was the first of the class of methods known currently as TWAS &hellip;
      <a href="/project/published-research/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>

  <section class="article-list normal-card">
    <table>
    <tbody>
    <tr>
    <td>
    
    <h1><a href="/project/ongoing-research/">Ongoing Research</a></h1>
    <div>
      
      
    </div>
    <div class="summary">
      
      
      
      
      &Popf;rediXcan method has proven to be successful and widely adopted. However there are several aspects that can be improved to increase detection and reduce false discoveries. A substantial portion of my effort will be devoted to developing methods to achieve this goal.
[&hellip;] Linkage disequilibrium (LD, correlation of genetic variation within proximal genomic regions) can lead to associated genes and phenotypes even when the underlying causal variants are distinct. This is a difficult problem given the widespread LD in the population. We are examining several approaches to mitigate this problem. One is by using fine-mapping methods, which seeks to increase the resolution of the association and narrow down the causal variant to a single variant. Another one is to use orthogonal approaches that seek to cancel out the confounding by averaging across multiple independent genes.
[&hellip;] The goal is to quantify the proportion of phenotypic variability that can be explained by the &hellip;
      <a href="/project/ongoing-research/" class="more" title="Read More...">Read more &rarr;</a>
    </div>
    </td>
    </tr>
    </tbody>
    </table>
  </section>










<section class="comments">
  <div id="disqus_thread"></div>
  <script>
  var disqus_config = function () {
  
  };
  (function() {
    var inIFrame = function() {
      var iframe = true;
      try { iframe = window.self !== window.top; } catch (e) {}
      return iframe;
    };
    if (inIFrame()) return;
    var disqus_js = '//hakyimlab.disqus.com/embed.js';
    var d = document, s = d.createElement('script');
    s.src = disqus_js; s.async = true;
    s.setAttribute('data-timestamp', +new Date());
    var t = d.getElementById('disqus_thread');
    var b = false, l = function(scroll) {
      if (b) return;
      (d.head || d.body).appendChild(s); b = true;
      if (scroll) t.scrollIntoView();
    }
    s.onerror = function(e) {
      if (sessionStorage.getItem('failure-note')) return;
      t.innerText = 'Sorry, but you cannot make comments because Disqus failed to load for some reason. It is known to be blocked in China. If you are sure it is not blocked in your region, please refresh the page. 中国大陆地区读者需要翻墙才能发表评论。';
      t.style.border = '1px dashed';
      t.style.padding = '.5em';
      t.style.background = 'lightyellow';
      sessionStorage.setItem('failure-note', true);
    };
    
    if (location.hash.match(/^#comment-[0-9]+$/)) return l(true);
    var c = function() {
      if (b) return;
      var rect = t.getBoundingClientRect();
      if (rect.top < window.innerHeight && rect.bottom >= 0) l();
    };
    window.addEventListener('load', c);
    d.addEventListener('scroll', c);
  })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</section>


</main>
</div>

  <footer>
  <script async src="//yihui.org/js/center-img.js"></script>





<script type="application/javascript">
var doNotTrack = false;
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'G-8SL8WKCMYZ', 'auto');
	
	ga('send', 'pageview');
}
</script>




  <div class="footer">
  
  <ul class="copyright">
    <li>© 2017 Hae Kyung Im</li>
  </ul>
  
  <ul>
    
    <li>
      <a href="/404.html">Contact</a>
    </li>
    
    <li class="optional">
      <a href="/categories/">Categories</a>
    </li>
    
    <li class="optional">
      <a href="/tags/">Tags</a>
    </li>
    
    <li id="menu-edit">
      <a href="#">Suggest an edit</a>
    </li>
    
    <li>
      <a href="#">Back to top</a>
    </li>
    
  </ul>
  </div>
  
  <div class="footer hidden">
  
<div>
  <div class="side side-left"><h3>Suggest changes</h3></div>
  
  
  
    
    
  
  If you find any mistakes (including typos) or want to suggest changes,
  please feel free to <a href="https://github.com/hakyimlab/web-homepageproject/_index.md" id="edit-link">edit
  the source file of this page</a> on Github and create a pull request.
</div>


  </div>
  
  </footer>
  <script async src="/js/features.js" data-page-features="[null,[&#34;&#43;sidenotes&#34;],null]"></script>
  </body>
</html>

